cloud_automation/gcf/airflow_trigger.py

```diff
@@ -0,0 +1,150 @@
+""" This module triggers when a file is writtent to prophet-cache GCS bucket.
+
+This create a cloud BT table, scales it up and then triggers a dataflow job.
+"""
+
+import google.auth
+import google.auth.compute_engine.credentials
+import google.auth.iam
+from google.auth.transport.requests import Request
+from google.cloud import storage
+import google.oauth2.credentials
+import google.oauth2.service_account
+import pathlib
+import requests
+
+
+IAM_SCOPE = 'https://www.googleapis.com/auth/iam'
+OAUTH_TOKEN_URI = 'https://www.googleapis.com/oauth2/v4/token'
+PROJECT_ID          = 'google.com:datcom-store-dev'
+DATAFLOW_TMPL   = 'gs://datcom-dataflow-templates/templates/csv_to_bt'
+PIPELINE_TRIGGER_FILE = 'airflow_trigger.txt'
+SUCCESS_FILE = 'success.txt'
+FAILURE_FILE = 'failure.txt'
+
+def read_contents(bucket, name):
+  client = storage.Client()
+  blob = client.get_bucket(bucket).blob(name)
+  result = blob.download_as_string()
+  return result
+
+
+def gcs_trigger(data, context=None):
```

Comment:
If this function errors out, where will it notify? Could you add some notes about it to http://go/dccachedocs.